src/marketing/buttons/button.scss

```
@@ -159,4 +159,6 @@
 .btn-large-mktg {
   // stylelint-disable-next-line primer/spacing
   padding: 20px 30px 23px !important;
```

The padding here differs from what is in Figma and causes the overall button height to be 63px instead of 56px. Here's what it should be adjusted to.
```suggestion
  padding: 16px 30px 20px !important;
```

---

❤️